pom.xml

```diff
@@ -640,9 +627,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.]]>
 				</configuration>
 			</plugin>
 			<plugin>
-				<groupId>org.codehaus.mojo</groupId>
+				<groupId>com.nickwongdev</groupId>
```

Comment:
org.codehaus.mojo isn't being maintained anymore and com.nickwongdev is now the defacto standard.  This is more future proof and com.nickwongdev supports java 9+ if we decide to go there which org.codehaus.mojo does not.